config/environments/production.rb

```diff
@@ -60,12 +60,7 @@
 
   config.action_controller.asset_host = ENV['GOVUK_ASSET_HOST']
 
-  if ENV['RUNNING_ON_HEROKU'].blank?
-    # Enable JSON-style logging
-    config.logstasher.enabled = true
-    config.logstasher.logger = Logger.new("#{Rails.root}/log/#{Rails.env}.json.log")
-    config.logstasher.suppress_app_logs = true
-  else
+  if ENV['RUNNING_ON_HEROKU']
```

Comment:
I think we might be able to skip this, because `govuk_app_config` will log to STDOUT by default?